src/main/java/com/google/sps/data/ApiKeys.java

```
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.BufferedReader;
+import java.util.stream.Collectors;
+
+public class ApiKeys {
+  public static final String MAPS_API_KEY = readKeyFromFile("/maps_api_key.txt");
+  public static final String WEATHER_API_KEY = readKeyFromFile("/openweather_api_key.txt");
+
+  private static String readKeyFromFile(String fileName) {
+    InputStream in = ApiKeys.class.getResourceAsStream(fileName);
+    if (in == null) {
+      System.err.println("Resource not found: " + fileName);
+      return "";
```

I would throw an exception here that the application fails to start without proper keys.